supplements/eberron-rising-from-the-last-war/class-artificer.xml

```diff
@@ -1644,4 +1650,111 @@
 			<select name="Feat (Level 19)" type="Feat" />
 		</rules>
 	</element>
-</elements>
\ No newline at end of file
+
+	<element name="Homunculus Servant" type="Companion" source="Eberron: Rising from the Last War" id="ID_WOTC_ERLW_COMPANION_ARTIFICER_HOMUNCULUS_SERVANT "> 
+		<supports>Homunculus Servant</supports>
+		<description> 
+			<p>You learn intricate methods for magically creating a special homunculus that serves you. The item you infuse (A gem or crystal worth at least 100 gp ) serves as the creature's heart, around which the creature's body instantly forms. </p> <p><b>Proficiency Bonus (PB)</b> equals your bonus</p> 
+		</description> 
+		<setters>
+			<set name="strength">4</set>
+			<set name="dexterity">15</set>
+			<set name="constitution">12</set>
+			<set name="intelligence">10</set>
+			<set name="wisdom">10</set>
+			<set name="charisma">7</set>
+			<set name="ac">13 (natural armor)</set>
+			<set name="hp">1 + your Intelligence modifier + your artificer level (the Homunculus has a number of Hit Dice [d4s] equal to your artificer level)</set>
+			<set name="speed">20 ft., fly 30 ft.</set>
+			<set name="immunities">poison</set>
+			<set name="conditionImmunities">exhaustion, poisoned</set>
+			<set name="saves">Dex +2 plus PB</set>
+			<set name="skills">Perception +0 plus PB x 2, Stealth +2 plus PB </set>
+			<set name="senses">darkvision 60 ft., passive Perception 10 + (PB x 2) </set>
+			<set name="languages">understands the languages you speak</set>
+			<set name="type">Construct</set>
+			<set name="size">Tiny</set>
+			<set name="alignment">neutral</set>
+			<set name="challenge">-</set>
+			<set name="traits">ID_WOTC_ERLW_COMPANION_TRAIT_HOMUNCULUS_SERVANT_EVASION</set>
+			<set name="actions">ID_WOTC_ERLW_COMPANION_ACTION_HOMUNCULUS_SERVANT_FORCE_STRIKE</set>
+			<set name="reactions">ID_WOTC_ERLW_COMPANION_REACTION_HOMUNCULUS_SERVANT_CHANNEL_MAGIC</set>
+		</setters> 
+		<rules>
+			<stat name="companion:ac" value="13" bonus="base" />
+			<stat name="companion:hp:max" value="1" />
+			<stat name="companion:hp:max" value="intelligence:modifier" />
+			<stat name="companion:hp:max" value="level:artificer" />
```

Comment:
i would change this in case someone would like to modify these values
```suggestion
			<stat name="companion:hp:max" value="1" bonus="base" />
			<stat name="companion:hp:max" value="intelligence:modifier" bonus="ability"/>
			<stat name="companion:hp:max" value="level:artificer" bonus="level"/>
```